src/Components/Forms/FormCreateUpdate.js

```diff
@@ -0,0 +1,316 @@
+import React from "react";
+import ButtonsCreate from "./ButtonsCreate";
+import ButtonsUpdate from "./ButtonsUpdate";
+import Input from "./Input";
+import ListItems from "./ListItems";
+import styles from "./FormCreateUpdate.module.css";
+import { GlobalContext } from "../../Context/GlobalMode";
+import { ReactComponent as ArrowLeft } from './../../Assets/icon-arrow-left.svg';
+
+const FormCreateUpdate = ({
+    typeForm,
+    address1,
+    address2,
+    city1,
+    city2,
+    code1,
+    code2,
+    country1,
+    country2,
+    name,
+    email,
+    invoiceDate,
+    project,
+    items,
+    setItems,
+    closeModal,
+    payment,
+    setPayment,
+    saveInvoice,
+    id,
+}) => {
+    const [error, setError] = React.useState(false);
+    const [error2, setError2] = React.useState(false);
+    const global = React.useContext(GlobalContext);
+
+    function validateForm() {
+        let validItems = false;
+        if (items.length === 0) {
+            setError2(true);
+        } else {
+            setError2(false);
+
+            const itemsValid = [...items];
+            itemsValid.forEach(({ name, qty, price }, index) => {
+                if (name.length === 0) itemsValid[index].nameError = true;
+                if (!/^\d+(\.\d{1,2})?$/.test(qty)) itemsValid[index].qtyError = true;
+                if (!/^\d+(\.\d{1,2})?$/.test(price))
+                    itemsValid[index].priceError = true;
+            });
+
+            validItems = itemsValid.every(({ nameError, qtyError, priceError }) => {
+                return !nameError && !qtyError && !priceError;
+            });
+
+            setItems(itemsValid);
+        }
+
+        if (
+            address1.validate() &&
+            address2.validate() &&
+            city1.validate() &&
+            city2.validate() &&
+            code1.validate() &&
+            code2.validate() &&
+            country1.validate() &&
+            country2.validate() &&
+            name.validate() &&
+            email.validate() &&
+            invoiceDate.validate() &&
+            project.validate() &&
+            validItems
+        ) {
+            setError(false);
+            return true;
```

Comment:
else ispod je suvisan jer ovde imas return 
```suggestion
            return true;
```